pkg/vdri/peer/vdri.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
looks like unit test is missing for this file. ie `vdri_test.go`

Reply:
it's already test it with other test file in the same package